docs/source/overview.md

```diff
@@ -0,0 +1,81 @@
+
+<figure style="display: table; text-align:center; margin-left: auto; margin-right:auto">
+
+![Sample linked data](./_static/img/mergedlinkeddata.svg)
+
+<figcaption style="display: table-caption; caption-side: bottom; text-align:center">
+
+_Linked data about a city from three different sources: the city's traffic authority, a map from a city guide, and the university registry. Each data source is represented using a different color and column._
+
+</figcaption>
+
+</figure>
+
+
+Moreover, in SimPhoNy, all linked data is based on [ontologies](https://en.wikipedia.org/wiki/Ontology_(information_science)), which give **meaning** to the data. Specifically, SimPhoNy works with ontologies based on the [Web Ontology Language](https://en.wikipedia.org/wiki/Web_Ontology_Language), making the data compatible with the [Semantic Web](https://en.wikipedia.org/wiki/Semantic_Web).
+
+### Fetch data from a database, run a simulation and immediately store the results
+
+Ontology-based linked data is not only well suited for the interoperability of data, but also of software tools. In SimPhoNy, one can instantiate individuals from special ontology classes called _wrappers_. These wrappers are both an ontology individual, and a software interface between the core of SimPhoNy (ontology based) and external software tools. We have already developed a few wrappers for some software tools, but you can even [develop your own](./wrapper_development.md)!
+
+As a SimPhoNy user, you can see the data stored in the external software tools transparently as ontology individuals through the wrappers. In this way, moving data between different software tools becomes as simple as moving or copying it from one wrapper to another.
+
+For example, linked data stored in a SQLite database can be used to run a simulation just by adding the ontology individuals contained in the SQLite wrapper to the Simulation Engine wrapper. Similarly, the ontology individuals representing the results can be simply added back into the database wrapper.
+
+<figure style="display: table; text-align:center; margin-left: auto; margin-right:auto">
+
+![How wrappers work](./_static/img/wrappers.svg)
+
+<figcaption style="display: table-caption; caption-side: bottom; text-align:center"></figcaption>
+
+</figure>
+
+At this point, the results could be fetched again and for example, visualized with the help of a plotting library.
+
+<figure style="display: table; text-align:center; margin-left: auto; margin-right:auto">
+
+![Toy example of simulation results](./_static/img/trafficsim.svg)
+
+<img src="./_static/img/overview_slider.png" width="350px">
+
+<figcaption style="display: table-caption; caption-side: bottom; text-align:center">
+
+<br>
+
+_These results are just a toy example, they do not come from any real nor factually correct simulation._
```

Comment:
Perhaps a caveat should be placed in the figure's caption

Reply:
This is already the figure's caption, note that it's inside the html caption tag. Is it sufficiently evident on the rendered docs? If the format of the caption needs to change, then I would have to further investigate other ways to put captions.

Reply:
Sorry - I meant should *not*.